test/mock/services/question.js

```diff
@@ -1,9 +1,11 @@
-/* eslint-disable max-len */
+/* eslint-disable max-len, no-confusing-arrow */
 module.exports = {
   path: '/continuous-online-hearings/:onlineHearingId/questions/:questionId',
+  method: 'GET',
   template: {
     question_id: params => `${params.questionId}`,
     question_header_text: 'How do you interact with people?',
-    question_body_text: 'You said you avoid interacting with people if possible. We\'d like to know more about the times when you see friends and family.\n\nTell us about three separate occasions in 2017 that you have met with friends and family.\n\nTell us:\n\n- who you met\n\n- when\n\n- where\n\n- how it made you feel'
+    question_body_text: 'You said you avoid interacting with people if possible. We\'d like to know more about the times when you see friends and family.\n\nTell us about three separate occasions in 2017 that you have met with friends and family.\n\nTell us:\n\n- who you met\n\n- when\n\n- where\n\n- how it made you feel',
+    question_answer_text: params => params.questionId === '1' ? 'My amazing answer' : ''
```

Comment:
Is this just used for testing what it looks like when an answer comes back? Just need to be not to add anything here that we can't replicate in the real service.